src/App/App.js

```diff
@@ -24,35 +24,32 @@ class App extends Component {
       { error: 'Something went wrong, please try again later.' }))
   }
 
-  goBacktoMainView = () => {
-    this.setState({ selectedMovie: null });
-  }
-
-  handleClick = (id) => {
-    getMovieDetails(id)
-      .then(data => {
-        this.setState({ selectedMovie: data.movie })
-      })
-      .catch(() => this.setState(
-        { error: 'Something went wrong, please try again later.' }))
-  }
-
   render() {
     return (
       <div className="container">
```

Comment:
This `<div>` can probably be removed now that everything has been nested inside the `<main>`, as it was only there to be a container.